src/features/user/TreeMapper/Import/components/SampleTrees.tsx

```diff
@@ -255,16 +255,24 @@ export default function SampleTrees({
       </div>
       <div className={`${styles.formField}`}>
         <div className={styles.formFieldHalf}>
-          <button onClick={handleSubmit(onSubmit)} className="primaryButton">
+          <button
+            onClick={handleSubmit(onSubmit)}
+            className="primaryButton"
+            style={{ width: '188px' }}
+          >
```

Comment:
1. Avoid using inline styles. It would be better to add styles in `Import.module.scss` to handle this styling. In this case, you can nest the button style inside the styles for `.formFieldHalf` along with `input` (which is already nested).
2. Please test your changes against different screen sizes (always!!). Currently, the buttons continue to overlap in mobile screens. I find it easier to style for mobile first, and then make adjustments for other screen sizes if needed.
3. Related to point 2 >>> It may be better to allow the button to take the full container width (100%) instead of specifying width = 188px. If you specify a pixel value for the width, you need to take care of other screen sizes as well by specifying a different pixel value. 

**Mobile: Width 100%**
![Screenshot 2023-01-23 at 10 33 37 AM](https://user-images.githubusercontent.com/44917347/213969660-edd51492-e664-424a-a704-3056f2faa378.png)

**Mobile: Width 188px**
![Screenshot 2023-01-23 at 10 41 17 AM](https://user-images.githubusercontent.com/44917347/213969896-6fac9c61-14c7-4a27-81b3-712c5ae37785.png)